spikeinterface/sortingcomponents/peak_localization.py

```diff
@@ -32,7 +32,7 @@ def init_kwargs_dict(method, method_kwargs, recording):
     if method_kwargs_.get("enforce_decrease", None) is not None:
         contact_locations = recording.get_channel_locations()
         channel_distance = get_channel_distances(recording)
-        neighbours_mask = channel_distance < method_kwargs['local_radius_um']
+        neighbours_mask = channel_distance < method_kwargs.get('local_radius_um', 150)
```

Comment:
I do not understand why. local_radius_um should be always there because it is done in init

Reply:
right, this logic is in the init, and sometimes this is not set at that point. could probably find a nicer way to structure it, should I give that a shot?

Reply:
I did patch a small stuff.